[2025-03-17T20:00:50.371+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T20:00:51.042+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_tables.pages_project_a_db scheduled__2025-03-17T19:50:00+00:00 [queued]>
[2025-03-17T20:00:51.302+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_tables.pages_project_a_db scheduled__2025-03-17T19:50:00+00:00 [queued]>
[2025-03-17T20:00:51.311+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-17T20:00:52.421+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): load_tables.pages_project_a_db> on 2025-03-17 19:50:00+00:00
[2025-03-17T20:00:53.353+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_tables.pages_project_a_db', 'scheduled__2025-03-17T19:50:00+00:00', '--job-id', '234', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp2472gpky']
[2025-03-17T20:00:53.561+0000] {standard_task_runner.py:105} INFO - Job 234: Subtask load_tables.pages_project_a_db
[2025-03-17T20:00:53.212+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3270) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-17T20:00:53.593+0000] {standard_task_runner.py:72} INFO - Started process 3294 to run task
[2025-03-17T20:00:55.542+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag.load_tables.pages_project_a_db scheduled__2025-03-17T19:50:00+00:00 [running]> on host c032966e105e
[2025-03-17T20:00:58.423+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load_tables.pages_project_a_db' AIRFLOW_CTX_EXECUTION_DATE='2025-03-17T19:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-17T19:50:00+00:00'
[2025-03-17T20:00:58.433+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-17T20:00:58.641+0000] {sql.py:278} INFO - Executing: select load_tables('pages_project_a_db', 'pages');
[2025-03-17T20:00:58.832+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T20:00:59.273+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T20:00:59.362+0000] {sql.py:553} INFO - Running statement: select load_tables('pages_project_a_db', 'pages');, parameters: None
[2025-03-17T20:00:59.351+0000] {job.py:229} INFO - Heartbeat recovered after 11.34 seconds
[2025-03-17T20:00:59.623+0000] {sql.py:562} INFO - Rows affected: 1
[2025-03-17T20:01:00.043+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-17T20:01:00.061+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load_tables.pages_project_a_db, run_id=scheduled__2025-03-17T19:50:00+00:00, execution_date=20250317T195000, start_date=20250317T200051, end_date=20250317T200100
[2025-03-17T20:01:00.331+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T20:01:00.582+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-17T20:01:00.602+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
